src/ui/components/UserReview/index.js

```diff
@@ -50,14 +59,18 @@ function reviewBody({
   );
 }
 
-const UserReview: React.ComponentType<Props> = ({
-  byLine,
-  children,
-  className,
-  controls,
-  review,
-  showRating = false,
-}: Props) => {
+export const UserReviewBase = (props: InternalProps) => {
+  const {
+    byLine,
+    children,
+    className,
+    controls,
+    i18n,
+    review,
+    showRating = false,
+    showDeveloperResponseHeading = false,
```

Comment:
The UI shows this text twice otherwise, see: https://github.com/mozilla/addons-frontend/pull/6285#pullrequestreview-156774504 (2)

Reply:
You're both correct. :-) I did need to hide it because it was being duplicated, but rather than special case it, I should just use the same layout/code for both views.